Project.toml

```diff
@@ -1,11 +1,13 @@
 name = "BlockArrays"
 uuid = "8e7c35d0-a365-5155-bbbb-fb81a777f24e"
-version = "0.11.2"
+version = "0.12"
```

Comment:
Out of curiosity, what did this version break? From this PR it looks like the old tests still work without any real change.

Reply:
` similar ` now sometimes returns a ` PseudoBlockArray ` where it used to return a ` BlockArray `. This is a breaking change for BlockBandedMatrices.jl